src/group_theory/p_group.lean

```diff
@@ -36,6 +36,9 @@ forall_congr (λ g, ⟨λ ⟨k, hk⟩, exists_imp_exists (by exact λ j, Exists.
 lemma of_card [fintype G] {n : ℕ} (hG : fintype.card G = p ^ n) : is_p_group p G :=
 λ g, ⟨n, by rw [←hG, pow_card_eq_one]⟩
 
```

Comment:
I would expect `is_p_group_bot` outside of this namespace.

Reply:
That sounds right to me too.

Reply:
Then again, this matches `of_card`...

Reply:
I suppose that a few of the lemmas could be renamed: `is_p_group_iff_order_of` `is_p_group_of_card` `is_p_group_bot` `is_p_group_iff_card`. But the extra verbosity isn't ideal.